operationsgateway_api/config.yml.example

```diff
@@ -20,6 +24,7 @@ mongodb:
   max_documents: 1000
   image_store_directory: /home/user/og-images
 auth:
+  # Create a private/public key pair for the auth functionality
```

Comment:
This should probably just say "Path to a private/public key pair for the auth functionality". 
For me (and possibly only me!) it is ambiguous whether "Create" is telling the user to create the keys or whether the application will be creating the keys at the specified location. Maybe a second line of comment is needed to clarify that the keys need to be created in advance and put at the specified location?